homeassistant/components/webostv/config_flow.py

```diff
@@ -204,7 +207,11 @@ async def async_get_sources(host: str, key: str) -> list[str]:
     except WEBOSTV_EXCEPTIONS:
         return []
 
-    return [
-        *(app["title"] for app in client.apps.values()),
-        *(app["label"] for app in client.inputs.values()),
-    ]
+    return list(
+        dict.fromkeys(
```

Comment:
We can use a set to deduplicate items.

Reply:
I thought about it, not sure if it is important, but set will change the order of the list, what do you think?

Reply:
Ok, then we should add a comment why we write it like this.